docs/source/geoSeries.rst

```diff
@@ -5,19 +5,19 @@ IdaGeoSeries
 The spatial methods of an IdaGeoDataFrame can be used to operate on the geometry attribute and wil return a IdaGeoSeries
 object in each case.
 An IdaGeoSeries is essentially a reference to a spatial column where each entry in the column is a
-set of shapes corresponding to one observation represented by DB2GSE.ST_GEOMETRY. An entry may consist of only one
+set of shapes corresponding to one observation represented by ST_GEOMETRY. An entry may consist of only one
 shape (like a ST_POINT/ST_LINESTRING/ST_POLYGON) or multiple shapes that are meant to be thought of as one observation
 (like the many polygons that make up the County of Santa Cruz in California or a state like Connecticut).
 
-Db2 has three basic classes of geometric objects, which are Db2 spatial objects that follow OGC guidelines:
+Netezza has three basic classes of geometric objects, which are Netezza spatial objects that follow OGC guidelines:
```

Comment:
We need geospatial wrappers as well. Current implementation is more of a db2 implementation and not Netezza specific.

Reply:
This file is not included in the doc build (that is, it will not be present in generated html). I can remove it if you want, but also we may leave it where it is for the future use when we finally manage to enable geoFrame/geoSeries modules for Netezza.